sos/plugins/openstack_cinder.py

```diff
@@ -45,6 +47,9 @@ def setup(self):
         else:
             self.add_copy_spec_limit("/var/log/cinder/*.log",
                                      sizelimit=self.limit)
+        if self.get_option("exclude"):
+            exclude = self.get_option("exclude")
+            self.add_forbidden_path(exclude)
```

Comment:
> just exclude the files and be done with it

The file names can be different for different deployments.

> Also, what's the point of setting the exclude variable here, since it's not used anywhere?

It's set on line 51 and used on line 52.